core/Config.php

```diff
@@ -678,77 +678,39 @@ public function remove($class, $name /*,$key = null*/ /*,$value = null*/) {
 
 }
 
+class Config_LRU extends ConfigCache {
+
+	public function __construct() {
+		Deprecation::notice('3.2', ' Config_LRU is deprecated, please use ConfigCache instead', Deprecation::SCOPE_CLASS);
+		parent::__construct();
+	}
+
+}
+
 /**
  * @package framework
  * @subpackage core
  */
-class Config_LRU {
-	const SIZE = 1000;
-
+class ConfigCache {
```

Comment:
Naming convention for any class in a file is that any part before the first underscore should match the name of the file.

`Config_MemCache` would be a better classname.